### PR TITLE
add LIMITATIONS.md

### PR DESCRIPTION
**Pull request by sjourdan**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #...
| ❓ Documentation  | yes 

## Description

We need a place to point our users to for known limitations., starting with the first we know about. 
It will keep getting better.

I placed it under `./doc` so it doesn't clutter the root of the repository and everything related to documentation is located at the same place.

### CODE REVIEW

File: doc/LIMITATIONS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Known Issues and Limitations
+
+## AWS Regions Limits
+
+- The user needs to use the same AWS region for both the scanned infrastructure and the S3 bucket where the Terraform state is stored (for example, a Terraform state stored on S3 on us-east-1 for an infrastructure to be scanned on us-west-1 won't work.).
+  - See the related [GitHub Discussion](https://github.com/cloudskiff/driftctl/discussions/130).
```
Comment (wbeuil) [inline doc/LIMITATIONS.md:6]: Can you remove the `.` inside the parentheses and aline the second `-`.

Thanks !

File: doc/LIMITATIONS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Known Issues and Limitations
+
+## AWS Regions Limits
+
+- The user needs to use the same AWS region for both the scanned infrastructure and the S3 bucket where the Terraform state is stored (for example, a Terraform state stored on S3 on us-east-1 for an infrastructure to be scanned on us-west-1 won't work.).
```
Comment (eliecharra) [inline doc/LIMITATIONS.md:5]: `the same AWS region` and the same credentials too !

**Review by eliecharra — CHANGES_REQUESTED** [review]
Maybe we should add a precision, actually we can only scan one provider, so for a state using multiples aws provider with aliases we cannot scan every provider.

We can handle multiples states but only one `--to` flag so only one remote.

**Review by eliecharra — APPROVED** [review]
LGTM

**Review by wbeuil — APPROVED** [review]

**Pull request merged**